src/message.ts

```diff
@@ -613,6 +625,40 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<any>
+  public forward(contact: Contact): Promise<any>
+  public forward(id: string): Promise<any>
+  public forward(sendTo: string|Room|Contact): Promise<any> {
```

Comment:
Should return `Promise<boolean>`, according to the `forward` function in `wechaty-bro.js`